add popcast (#26)
cilium/cilium.io: src/posts/2020-10-21-popcast/index.md

---
date: '2020-10-21T17:00:00.000Z'

> title: 'Episode 40 CNI and EBPF in Kubernetes with Cilium's Thomas Graf'

Author: ```suggestion
title: 'Episode 40 CNI and EBPF in Kubernetes with Thomas Graf from Cilium'
```

> title: 'Episode 40- CNI and EBPF in Kubernetes with Cilium's Thomas Graf'

Author: ```suggestion
title: 'Episode 40 CNI and EBPF in Kubernetes with Cilium's Thomas Graf'
```

> ogImageUrl: 'https://pbs.twimg.com/profile_banners/1256677532834938883/1588450683/'

Author: ```suggestion
ogImageUrl: 'https://twitter.com/PopcastPop/header_photo'
```

> ogImageUrl: 'https://pbs.twimg.com/profile_banners/1256677532834938883/1588450683/1500x500'

Author: ```suggestion
ogImageUrl: 'https://pbs.twimg.com/profile_banners/1256677532834938883/1588450683/'
```

ogSummary: 'Thomas helps us understand better the benefits of eBPF and the open source projects Cilium along with Hubble'
categories:

>   - Technology

Author: ```suggestion
  - Community
```
